src/main/java/com/thed/utils/ParserUtil.java

```diff
@@ -227,6 +260,13 @@ Map deepCopy(Map originalMap) {
         return cloneMap;
     }
 
+    List deepyCopy(List originalList) {
+        Gson gson = new Gson();
+        String json = gson.toJson(originalList);
+        List cloneList = gson.fromJson(json, List.class);
```

Comment:
This deserializes into a new instance is it?

Reply:
Yes, it creates deep copy without any relation to original.